code/game/objects/items/storage/secure.dm

```diff
@@ -61,6 +62,20 @@
 	// -> storage/attackby() what with handle insertion, etc
 	return ..()
 
+/obj/item/storage/secure/emag_act(mob/user)
+	user.show_message("<span class='danger'>Now attempting to scramble internal memory.</span>", 1)
+	src.l_hacking = TRUE

```

Comment:
Remove `src.` from everything with it

Reply:
Yea, I copied code from the multitool hack, changed both now.